CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ Note: This is in reverse chronological order, so newer entries are added to the
 Swift 5.8
 -----------
 
+* [#5949]
```

Comment:
I think we should sort these

Reply:
It's already in reverse chronological order, or would you like to see some other sorting?

Reply:
I think the rest of the document has is non-reverse chronological order?